README.md

```diff
@@ -792,6 +792,7 @@ Table of Contents
    * [Quickmetrics](https://www.quickmetrics.io/) — Timeseries database with dashboard included, free up to 10,000 events/day and total of 5 metrics.
    * [restdb.io](https://restdb.io/) - a fast and simple NoSQL cloud database service. With restdb.io you get schema, relations, automatic REST API (with MongoDB-like queries) and an efficient multi-user admin UI for working with data. Free plan allows 3 users, 2500 records and 1 API requests per second.
    * [Unbounded](https://www.unbounded.cloud/) - a NoSQL DBaaS which uses Javascript as a query language. Free tier allows 50,000 reads and writes per month, forever.
+    * [Freedb](https://freedb.tech) - Free MySQL cloud database with remote access up to 100Mb in Database Storage with PhpMyAdmin available and unlimited simultaneous connections. 
```

Comment:
the free plan is not unlimited connections. It also says `The database exceeding 100Mb will be deleted without any warning.` which I would say is just unbelievable and so I dont think I care to add it to the list

Reply:
I agree with you. However, it is good for a quick test for on development, process the request quickly and compared to `https://www.db4free.net/` I found it better. 

I would say add it with a warning note and deleting the `unlimited simultaneous connections.`?

Reply:
OK, lets do that